Plutarch/Api/V1/AssocMap.hs

```diff
@@ -3,12 +3,41 @@
 
 module Plutarch.Api.V1.AssocMap (
   PMap (PMap),
+
+  -- * Creation
+  empty,
```

Comment:
Naming

Reply:
You mean you want `pempty` instead? Any other functions to put p on?